devfiles/apache-camel-springboot/devfile.yaml

```diff
@@ -16,9 +16,11 @@ components:
   -
     type: chePlugin
     id: camel-tooling/vscode-apache-camel/latest
+    memoryLimit: 128Mi
```

Comment:
I suspect too-low limits are the cause the tcp failure. Che server pings services to determine when all components in a workspace are up and running, and a sidecar that fails due to bumping up against a too-low limit will prevent workspace start (in some cases).

We're generally avoiding setting memory limits in devfiles since (ideally) the plugin registry should provide a minimum amount required for each plugin (although I understand why it's being done for this devfile).

Reply:
you were right, i set it to 200M and it is working